runtimes/web/src/framebuffer.js

```diff
@@ -98,50 +171,102 @@ export class Framebuffer {
-                this.drawPointUnclipped(strokeColor, x0 + x, y0 - y); /*  II. Quadrant */
-                this.drawPointUnclipped(strokeColor, x0 - x, y0 + y); /* III. Quadrant */
-                this.drawPointUnclipped(strokeColor, x0 - x, y0 - y); /*  IV. Quadrant */
-
-                y++; sy += aa2; e += dy; dy += aa2;
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 + x,
+                    y0 + y
+                ); /*   I. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 + x,
+                    y0 - y
+                ); /*  II. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 - x,
+                    y0 + y
+                ); /* III. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 - x,
+                    y0 - y
+                ); /*  IV. Quadrant */
+
+                y++;
+                sy += aa2;
+                e += dy;
+                dy += aa2;
                 if (2 * e + dx > 0) {
-                    x--; sx -= bb2; e += dx; dx += bb2;
+                    x--;
+                    sx -= bb2;
+                    e += dx;
+                    dx += bb2;
                 }
             }
         }
 
         {
-            let x = 0, y = b;
-            let dx = b * b, dy = (1 - 2 * b) * a * a;
-            let sx = 0, sy = aa2 * b;
+            let x = 0,
+                y = b;
+            let dx = b * b,
+                dy = (1 - 2 * b) * a * a;
+            let sx = 0,
+                sy = aa2 * b;
             let e = 0;
 
             while (sy >= sx) {
-                this.drawPointUnclipped(strokeColor, x0 + x, y0 + y); /*   I. Quadrant */
-                this.drawPointUnclipped(strokeColor, x0 + x, y0 - y); /*  II. Quadrant */
-                this.drawPointUnclipped(strokeColor, x0 - x, y0 + y); /* III. Quadrant */
-                this.drawPointUnclipped(strokeColor, x0 - x, y0 - y); /*  IV. Quadrant */
-
-                x++; sx += bb2; e += dx; dx += bb2;
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 + x,
+                    y0 + y
+                ); /*   I. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 + x,
+                    y0 - y
+                ); /*  II. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 - x,
+                    y0 + y
+                ); /* III. Quadrant */
+                this.drawPointUnclipped(
+                    strokeColor,
+                    x0 - x,
+                    y0 - y
+                ); /*  IV. Quadrant */
+
+                x++;
+                sx += bb2;
+                e += dx;
+                dx += bb2;
```

Comment:
this is really controversial formatting in my opinion

Reply:
> this is really controversial formatting in my opinion

Formatting tools like prettier, gofmt, rustfmt, deno fmt where invented to save time on seldom productive aesthetic discussions.

We should probably run  a formatting tool, e.g. prettier,  on pre-commit.

Reply:
I mean at least --print-width could be increased to 90 instead 80 for avoiding such wraps. Also will be great add prettify in separate PR

Reply:
Then, if you ran a prettier, you should unroll variable declarations manually, and replace a , onto ;
becuase a blocks like:
```
const a = 10,
    b = 20,
    c = 3232
```

I less readable that `const a = 10, b = 20, c = 3232`

Reply:
+1 to leaving the reformatting to another PR.

I don't care much about code formatting, but `prettier` (at least with default settings) seems too strict and ends up with degenerate cases like in this PR:

```js
                    this.blit(
                        FONT,
                        currentX,
                        y,
                        8,
                        8,
                        0,
                        (charCode - 32) << 3,
                        8
                    );
```

For what it's worth I agree the formatting of some parts of `framebuffer.js` are fugly, and that's on me for copy-pasting from other projects or porting from C without fixing up the formatting after :sweat_smile: